charts/tfy-gpu-operator/templates/fractional-gpu-timeslicing.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.clusterType.azureAks }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: time-slicing-config
```

Comment:
Lets not hardcode but make it a variable and pick from values. Same for the number of replicas at line #15